src/tcgwars/logic/impl/gen4/SupremeVictors.groovy

```diff
@@ -229,378 +245,519 @@ public enum SupremeVictors implements LogicCardInfo {
+                  clearSpecialCondition(it)
+                }
+              }
             }
           }
           move "Desperate Pollen", {
             text "30 damage. If Venusaur already has 8 or more damage counters on it, the Defending Pokémon is now Burned, Confused, and Poisoned."
             energyCost G
             attackRequirement {}
             onAttack {
-              damage 0
+              damage 30
+              afterDamage {
+                if(self.numberOfDamageCounters() >= 8) {
+                  apply BURNED
+                  apply CONFUSED
+                  apply POISONED
+                }
+              }
             }
           }
           move "Special Reaction", {
             text "40+ damage. If the Defending Pokémon is affected by any Special Conditions, this attack does 40 damage plus 40 more damage for each of those Special Conditions."
             energyCost G, G, C, C
             attackRequirement {}
             onAttack {
-              damage 0
+              damage 40 + 40 * defending.specialConditions.size()
             }
           }
-
         };
       case YANMEGA_14:
-        return evolution (this, from:"Yanma", hp:HP100, type:GRASS, retreatCost:0) {
-          weakness L
+        return evolution (this, from:"Yanma", hp:HP100, type:G, retreatCost:0) {
+          weakness L, PLUS20
           resistance F, MINUS20
           pokePower "Speed Boost", {
-            text "Once during your turn , if Yanmega is your Active Pokémon, you may search your discard pile for a Energy card and attach it to Yanmega. This power can’t be used if Yanmega is affected by a Special Condition."
+            text "Once during your turn (before your attack), if Yanmega is your Active Pokémon, you may search your discard pile for a [G] Energy card and attach it to Yanmega. This power can't be used if Yanmega is affected by a Special Condition."
             actionA {
+              checkLastTurn()
+              checkNoSPC()
+              assert self.active : "$self is not your active Pokémon"
+              assert my.discard.filterByEnergyType(G) : "You have no [G] Energy cards in your discard pile"
+              powerUsed()
+              attachEnergyFrom(type:G,my.discard,self)
             }
           }
           move "Wind Return", {
-            text "20+ damage. Energy attached to Yanmega to your hand. If you do, this attack does 20 damage plus 20 more damage for each Energy card you returned."
-            energyCost C, C, G
+            text "20+ damage. You may return all [G] Energy attached to Yanmega to your hand. If you do, this attack does 20 damage plus 20 more damage for each Energy card you returned."
+            energyCost C, C
             attackRequirement {}
             onAttack {
-              damage 0
+              damage 20
+              if(confirm("Return all [G] Energy attached to $self in order to do 20 additional damage?")) {
+                damage 20
```

Comment:
This doesn't return the cards, nor multiply by the number of cards returned to the hand.